examples/amp-lightbox.amp.html

```diff
@@ -35,19 +35,24 @@
       z-index: 10;
     }
   </style>
-  <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-0.1.js"></script>
+  <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-1.0.js"></script>
```

Comment:
Are there some examples not covered by the storybooks?

Reply:
The storybooks are all displayed in an iframe which obscures the scroll containment behavior (gives false positives for blocking) whereas amp-lightbox in just a regular document such as this example lets us confirm the behavior outside of iframe.

Reply:
I see. Unfortunately, even "Canvas in new tab" mode still uses an iframe :(